daemon/container.go

```diff
@@ -1137,6 +1138,31 @@ func (container *Container) shouldRestart() bool {
 		(container.hostConfig.RestartPolicy.Name == "on-failure" && container.ExitCode != 0)
 }
 
+func (container *Container) mountVolumes() error {
```

Comment:
There used to be, but there was a recent `volumes` refactor (#13161 by @calavera) and so I added this method back in.